DigitalEcosystem/refined/metal_classifier/metalClassifier.py

```diff
@@ -157,7 +156,7 @@ def keep_best_bg(study, trial):
 def objective(trial: optuna.Trial):
     global current_pipeline
 
-    tr_x, val_x, tr_y, val_y = sklearn.model_selection.train_test_split(res_x, res_y, test_size=0.2)
+    tr_x, val_x, tr_y, val_y = sklearn.model_selection.train_test_split(res_x, res_y, test_size=0.2, random_state=RANDOM_SEED)
```

Comment:
Let's spell the variable names, pls: tr -> train, val -> value, etc.

Reply:
Increased verbosity of the naming convention